Tasks/Palishchuk/task4.py

```diff
@@ -0,0 +1,61 @@
+while True:
```

Comment:
I would suggest not to place all the logic inside the cycle, only the validation-related one